src/Mod/Sketcher/Gui/TaskSketcherElements.cpp

```diff
@@ -498,6 +498,7 @@ TaskSketcherElements::TaskSketcherElements(ViewProviderSketch *sketchView)
     , previouslyHoveredItemIndex(-1)
     , previouslyHoveredType(SubElementType::none)
     , isNamingBoxChecked(false)
+    , collapseFilter(true)

```

Comment:
Inconsistent to initialize to `true`

Reply:
Why? Does that change anything? Default value is true, so I initialized to true.

Reply:
Hmmm in `TaskSketcherElements::createSettingsButtonActions(` : 
`collapseFilter = hGrp->GetBool("AutoCollapseFilter", false);`